src/tile.js

```diff
@@ -264,10 +264,22 @@ $.Tile = function(level, x, y, bounds, exists, url, context2D, loadWithAjax, aja
     this._wcKey = `w${_workingCacheIdDealer++}://` + this.originalCacheKey;
     /**
      * Processing flag, exempt the tile from removal when there are ongoing updates
-     * @member {Boolean}
+     * @member {Boolean|Number}
      * @private
      */
     this.processing = false;
+    /**
+     * Remembers last processing time of the tile, 1 if the tile has just been loaded.
+     * @private
+     */
+    this.lastProcess = 0;
+    /**
+     * Transforming flag, exempt the tile from any processing since it is being transformed to a drawer-compatible
+     * format. This process cannot be paused and the tile cannot be touched during the process. Used externally.
```

Comment:
What does "used externally" mean in this context? I see it's marked as private, so it's not part of the public API, so I guess it means it's used by other components within OSD?